test/bdd/features/did_method_e2e.feature

```diff
@@ -10,10 +10,11 @@ Feature: Using DID method REST API
 
   @e2e
   Scenario Outline: create trustbloc did and resolve it through
-    Given TrustBloc DID is created through registrar "http://localhost:9080/1.0/register?driver-did-method-rest" with key type "<keyType>"
-    Then Resolve created DID through resolver URL "http://localhost:8080/1.0/identifiers" and validate key type "<keyType>"
+    Given TrustBloc DID is created through registrar "http://localhost:9080/1.0/register?driver-did-method-rest" with key type "<keyType>" with signature suite "<signatureSuite>"
+    Then Resolve created DID through resolver URL "http://localhost:8080/1.0/identifiers" and validate key type "<keyType>", signature suite "<signatureSuite>"
     Examples:
-      | keyType  |
-      | Ed25519  |
-      | EC       |
-      |          |
+      | keyType  |  signatureSuite             |
+      | Ed25519  |  JwsVerificationKey2020     |
+      | EC       |  JwsVerificationKey2020     |
+      | Ed25519  |  Ed25519VerificationKey2018 |
+      | EC       |  Ed25519VerificationKey2018 |
```

Comment:
Does EC key work with Ed25519VerificationKey2018 ?

Reply:
yes

Reply:
@rolsonquadras I will remove it